lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/client/LocalJobSubmissionHandler.java

```diff
@@ -108,10 +110,22 @@ public void onNext(final JobSubmissionEvent t) {
         this.configurationSerializer.toFile(driverConfiguration,
             new File(driverFolder, this.fileNames.getDriverConfigurationPath()));
         this.driverLauncher.launch(driverFolder);
+        this.applicationId = t.getIdentifier();
```

Comment:
Why did you add a variable `appId` to hold application id in `HDInsightJobSubmissionHandler.java` but not in this file or in Mesos one?